Tests/test_PDB_PSEA.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 by Valentin Vareskic (valentin.vareskic@gmail.com).
+# All rights reserved. This code is part of the Biopython distribution
+# and governed by its license.  Please see the LICENSE file that should
+# have been included as part of this package.
+
+"""Tests for PDB PSEA."""
+
+import unittest
+from Bio.PDB.PSEA import run_psea, psea, psea2HEC, PSEA
+from Bio.PDB import PDBParser
+from subprocess import getoutput
+from Bio import MissingExternalDependencyError
+import os
+
+if "command not found" or "'psea' is not recognized" in getoutput("psea -h"):
```

Comment:
As in the examples referred to, please add this line above:

```
os.environ['LANG'] = 'C'
```

We want to ensure the error messages are in US English to match the if-statement.